tests/InstrEngineTests/TestScripts/MultiReturn_IfTest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<InstrumentationTestScript>
+  <InstrumentMethod>
+    <ModuleName>MultiReturnTests_Release_x64.exe</ModuleName>
+    <MethodName>MultiIfTest</MethodName>
+    <!-- reset to single return -->
```

Comment:
><!-- reset to single return --> [](start = 4, length = 31)

nit remove unless this is important for the MultiIfTest?

Reply:
Yeah, I was originally going to add instrumentation, and then do the single return after that, but it ended up being more complicated than necessary. I'll remove this.

---
In reply to: [522466577](https://github.com/microsoft/CLRInstrumentationEngine/pull/326#discussion_r522466577) [](ancestors = 522466577)